test/plugins/test-trace-node-fetch.ts

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as fetchModule from 'node-fetch'; // For types only.
```

Comment:
nit: for cases where a module is imported for types only, it is better to name it `fetchTypes`. This helps accidental, non-type use, in future edits.

Reply:
I went with `fetchModule` as `typeof fetchModule` reads better than `typeof fetchTypes`. But either works, so I've changed it.